torch/csrc/jit/tensorexpr/expr.cpp

```diff
@@ -90,6 +90,14 @@ Expr exp(const Expr& v) {
   return Intrinsics::make(kExp, v);
 }
 
+Expr expm1(const Expr& v) {
+  return Intrinsics::make(kExpm1, v);
+}
+
+Expr neg(const Expr& v) {
+  return Intrinsics::make(kNeg, v);
```

Comment:
`neg` in the current fuser is `-${0}`, and llvm doesn't appear to have an intrinsic for this...

Reply:
`neg` shouldn't be an intrinsic, IMO.